### PR TITLE
Improve instructions for using local repositories

### PR DESCRIPTION
**Pull request by guangyee**
Add want_cached_images environment variable to the sample script.

### CODE REVIEW

File: docs/mkcloud.md [hunk 1]
```diff
@@ -150,19 +150,38 @@ A basic working mkcloud environment could look like [this](basic-mkcloud-config.
...
 
-To be able to deploy a complete Cloud with `mkcloud` and without network access,
-you can run the "prepare" step in mkcloud using the cache_clouddata=1 environment
-variable set. This will create a cache under $cache_dir (set to /var/cache/mkcloud by default
-of images and repositories needed during deployment.
+To be able to deploy a complete Cloud with `mkcloud` and without VPN access,
+you can run the "prepare" step in mkcloud using the `cache_clouddata=1`
+environment variable set. This will create a cache under `$cache_dir` (set to
+`/var/cache/mkcloud` by default) of images and repositories needed during
+deployment. Make sure your `$cache_dir` partition have enough free space to
```
Comment (nicolasbock) [inline docs/mkcloud.md]: s/have/has/

File: docs/mkcloud.md [hunk 2]
```diff
@@ -150,19 +150,38 @@ A basic working mkcloud environment could look like [this](basic-mkcloud-config.
...
 #!/bin/bash
 
-# path to the locally available repositories
+# path to the locally available repositories. If you are using the default
+# cache_dir (i.e. did not explicitly set the cache_dir environment
+# variable, localreposdir should point to root of cached repos.
+# i.e. export localreposdir_src=/var/cache/mkcloud/cloud/
+# Otherwise, localreposdir_src should be pointing to your cache_dir
+# environment variable. i.e.
+# export localreposdir_src=$cache_dir
```
Comment (dirkmueller) [inline docs/mkcloud.md]: can you change this to 
```
export cache_clouddata=1
# export cache_dir=<someotherdir>
```

File: docs/mkcloud.md (hunk 1 above)
Comment (guangyee) [inline docs/mkcloud.md]: good catch!

File: docs/mkcloud.md (hunk 2 above)
Comment (guangyee) [inline docs/mkcloud.md]: Thanks! Yes I will make the update.

**Pull request merged**